Application/Commands/Admin/ProcessEmailsCommand.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Application.Common.Interfaces;
+using Application.Common.Models.Admin;
+
+namespace Application.Commands.Admin;
+
+public record ProcessEmailsCommand : IRequest<ProcessedEmailsModel>
+{
+    public string? NotificationId { get; set; } = null!;
```

Comment:
Do we need this?

Reply:
Good shout, old testing code that has no impact but certain should have been removed.  Done.